### ISSUE TITLE
`Deno.emit` hangs when setting `module` to true or false

### ISSUE THREAD

**Issue by ebebbington** [denoland/deno#11198]
Deno version: 1.11.2

```ts
const { diagnostics, files } = await Deno.emit("./components/App.ts", {
    bundle: "module", // or "common", either one hangs
    compilerOptions: {
        "target": "es2020",
        "module": "esnext",
        "lib": ["dom", "DOM.Iterable", "ESNext"],
        "sourceMap": false
    },
})
```

When the `bundle` property is defined in the emit optionsl the process hangs. This might be due to hanging because of a circular dependency (which does exist in the code base im using), so it might be similar to https://github.com/denoland/deno/issues/8224

**Comment by kitsonk** [comment]
What does `deno bundle --config tsconfig.json ./components/App.ts` with the settings moved into a `tsconfig.json` do?

**Comment by ebebbington** [comment]
@kitsonk hangs, regardless of the contents of the tsconfig - `App.ts` uses a library that has a circular dependency and due to this, no matter what settings I use when using `Deno bundle`, it will hang (related issue is in this issues description)

**Comment by ebebbington** [comment]
Fixed by upgrading `swc_bundler` to `0.45.0`, closing as this is resolved
